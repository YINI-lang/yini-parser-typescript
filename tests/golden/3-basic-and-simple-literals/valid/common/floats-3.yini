^ Numbers
float9  = -9.999e+99         		// Large negative float, scientific
--float10 = 0.000000000000000000001  	// Smallest normal value you expect to support
--float11 = -0.000000000000000000001 	// Negative small
float12 = 2.2250738585072014e-308  	// Smallest positive normal double
float13 = -1.7976931348623157e+308 	// Min (negative) JS double
