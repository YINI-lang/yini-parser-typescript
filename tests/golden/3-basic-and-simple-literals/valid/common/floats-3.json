{
    "Numbers": {
        "float9": -9.999e+99,
        "float12": 2.2250738585072014e-308,
        "float13": -1.7976931348623157e+308
    }
}
